app/views/faq_groups/basics.md

```diff
@@ -1,37 +1,36 @@
 # How does Assembly work?
 
-Assembly enables products from software and hardware, to online communities, to be built like open source projects. Contributors earn a part of the product (in App Coins) when they complete a bounty made by the core team of a product.
+Assembly enables products--from software and hardware, to online communities--to be built like open source projects. Contributors earn a part of a product (in App Coins) when they complete a bounty made by that product's core team.
```

Comment:
You should replace “- -” with a real emdash: `—` (read more [here](http://typogui.de)). Also, spaces should be used around emdashes :)